packages/components/src/components/PhoneInput/constants.ts

```diff
@@ -255,503 +258,273 @@ type Countries = {
 // Modifisert til å passe våre behov: nummer som int
 
 export const COUNTRIES: Countries = {
-  AF: {
-    name: 'Afghanistan (‫افغانستان‬‎)',
-    id: 'AF',
-    dialCode: '+93',
-  },
-  AL: {
-    name: 'Albania (Shqipëri)',
-    id: 'AL',
-    dialCode: '+355',
-  },
-  DZ: {
-    name: 'Algeria (‫الجزائر‬‎)',
-    id: 'DZ',
-    dialCode: '+213',
-  },
+  AF: { name: 'Afghanistan (‫افغانستان‬‎)', id: 'AF', dialCode: '+93' },
+  AL: { name: 'Albania (Shqipëri)', id: 'AL', dialCode: '+355' },
+  DZ: { name: 'Algerie (‫الجزائر‬‎)', id: 'DZ', dialCode: '+213' },
   AS: {
-    name: 'American Samoa',
+    name: 'Amerikansk Samoa (American Samoa)',
     id: 'AS',
     dialCode: '+1684',
   },
   AD: { name: 'Andorra', id: 'AD', dialCode: '+376' },
   AO: { name: 'Angola', id: 'AO', dialCode: '+244' },
   AI: { name: 'Anguilla', id: 'AI', dialCode: '+1264' },
   AG: {
-    name: 'Antigua and Barbuda',
+    name: 'Antigua og Barbuda (Antigua and Barbuda)',
     id: 'AG',
     dialCode: '+1268',
   },
   AR: { name: 'Argentina', id: 'AR', dialCode: '+54' },
-  AM: {
-    name: 'Armenia (Հայաստան)',
-    id: 'AM',
-    dialCode: '+374',
-  },
+  AM: { name: 'Armenia (Հայաստան)', id: 'AM', dialCode: '+374' },
   AW: { name: 'Aruba', id: 'AW', dialCode: '+297' },
   AU: { name: 'Australia', id: 'AU', dialCode: '+61' },
-  AT: {
-    name: 'Austria (Österreich)',
-    id: 'AT',
-    dialCode: '+43',
-  },
-  AZ: {
-    name: 'Azerbaijan (Azərbaycan)',
-    id: 'AZ',
-    dialCode: '+994',
-  },
+  AT: { name: 'Østerrike (Österreich)', id: 'AT', dialCode: '+43' },
+  AZ: { name: 'Aserbajdsjan (Azərbaycan)', id: 'AZ', dialCode: '+994' },
   BS: { name: 'Bahamas', id: 'BS', dialCode: '+1242' },
-  BH: {
-    name: 'Bahrain (‫البحرين‬‎)',
-    id: 'BH',
-    dialCode: '+973',
-  },
-  BD: {
-    name: 'Bangladesh (বাংলাদেশ)',
-    id: 'BD',
-    dialCode: '+880',
-  },
+  BH: { name: 'Bahrain (‫البحرين‬‎)', id: 'BH', dialCode: '+973' },
+  BD: { name: 'Bangladesh (বাংলাদেশ)', id: 'BD', dialCode: '+880' },
   BB: { name: 'Barbados', id: 'BB', dialCode: '+1246' },
-  BY: {
-    name: 'Belarus (Беларусь)',
-    id: 'BY',
-    dialCode: '+375',
-  },
+  BY: { name: 'Hviterussland (Беларусь)', id: 'BY', dialCode: '+375' },
   BE: { name: 'Belgium (België)', id: 'BE', dialCode: '+32' },
   BZ: { name: 'Belize', id: 'BZ', dialCode: '+501' },
   BJ: { name: 'Benin (Bénin)', id: 'BJ', dialCode: '+229' },
   BM: { name: 'Bermuda', id: 'BM', dialCode: '+1441' },
   BT: { name: 'Bhutan (འབྲུག)', id: 'BT', dialCode: '+975' },
   BO: { name: 'Bolivia', id: 'BO', dialCode: '+591' },
   BA: {
-    name: 'Bosnia and Herzegovina (Босна и Херцеговина)',
+    name: 'Bosnia og Hercegovina (Босна и Херцеговина)',
```

Comment:
Ser at det norske navnet er Bosnia-Hercegovina ref.: https://snl.no/Bosnia-Hercegovina
```suggestion
    name: 'Bosnia-Hercegovina (Босна и Херцеговина)',
```